funding/manager.go

```diff
@@ -1818,10 +1818,8 @@ func (f *Manager) fundeeProcessOpenChannel(peer lnpeer.Peer,
 	//
 	// NOTE: Even with this bounding, the ChannelAcceptor may return an
 	// BOLT#02-invalid ChannelReserve.
-	maxDustLimit := reservation.OurContribution().DustLimit
-	if msg.DustLimit > maxDustLimit {
-		maxDustLimit = msg.DustLimit
-	}
+	maxDustLimit := max(msg.DustLimit,
```

Comment:
https://github.com/lightningnetwork/lnd/blob/master/docs/development_guidelines.md#wrapping-long-function-calls

Reply:
Oh. Thanks.  

Changed. Please review again.

Reply:
It's still wrong in multiple places.

Reply:
Sorry, but I've checked, are you referring to this? https://github.com/lightningnetwork/lnd/pull/9803/files#diff-366e46a40f6f60b4f7614eb0976bb51820364bf5ca6ccc4787eb49d7bdbef3e6L1617. 

However, this follows the previous style; does it need to be modified?

Reply:
No, I mean how to format a function call that's too long.
The correct way would be:
```
		delay := max(
				uint16(btcutil.Amount(maxRemoteDelay)* chanAmt/MaxFundingAmount),
				minRemoteDelay,
		)
```

Although I think in this case for readability it should be:
```
		delay := uint16(btcutil.Amount(maxRemoteDelay)* chanAmt/MaxFundingAmount)
		delay = max(delay, minRemoteDelay)
```

Reply:
Only the overlong code needs to be modified, and no action is required for changes that do not exceed the length limit. According to this rule, I see no errors. 

Could you provide further suggestions?

Reply:
> No, I mean how to format a function call that's too long. The correct way would be:
> 
> ```
> 		delay := max(
> 				uint16(btcutil.Amount(maxRemoteDelay)* chanAmt/MaxFundingAmount),
> 				minRemoteDelay,
> 		)
> ```
> 
> Although I think in this case for readability it should be:
> 
> ```
> 		delay := uint16(btcutil.Amount(maxRemoteDelay)* chanAmt/MaxFundingAmount)
> 		delay = max(delay, minRemoteDelay)
> ```

Sorry, I just saw your reply. However, the line `uint16(btcutil.Amount(maxRemoteDelay)*chanAmt/MaxFundingAmount)` already exceeds 80 characters with several tabs added at the beginning, so it must be wrapped.

Reply:
I have made the changes. Please review again.

Reply:
No, you have not, there's still one missing. If you're unable to follow simple instructions and/or look at your commit before re-requesting review, you're just wasting my time. Bye.

Reply:
I will check after submitting. I didn't expect your reply to be so quick; it was processed in less than a minute after my submission, so I hadn't had the chance to do so (I usually rely on GitHub's visual interface for self-review after submitting).